src/Service.Tests/Unittests/MultiSourceQueryExecutionUnitTests.cs

```diff
@@ -41,6 +48,13 @@ public class MultiSourceQueryExecutionUnitTests
                 }
             }";
 
+        private const string DATASOURCENAME1 = "db1";
```

Comment:
nit: underscore should separate words for constants, ie: DATA_SOURCE_NAME_1

Reply:
fixed.